### DISCUSSION TITLE
Any idea why some channels are missing after reimporting into LG TV?

### DISCUSSION THREAD

**Discussion by matthiasrebbe** [PredatH0r/ChanSort#331] [Q&A]
Hi,
maybe there is someone with a similar problem.
On my LG 50PB560U (DVBS2/Astra) I've made a full channel search. I did not use a specific service provider (Astra / Sky).
I've then exported the channel list and used ChanSort to sort the channels. I then reimported the new channellist to the LG, but for whatever reason channel 1-3 (Das Erst HD, ZDF HD, WDR HD Bielefeld) are missing. The list starts with channel 4. And there are some other channels missing. When i export the channel list again from LG to USB and read that new list into ChanSort, then all missing channels are shown/listed.
Does anyone have an idea, what might be the problem?

**Comment by PredatH0r** [comment]
Hello Matthias,

I have heard about various random issues with webOS 6 TVs and my only explanation is that there are many bugs in the firmware related to channel list transfers.
The cleanest way to import a list is to reset the TV first, skip the channel search (using EXIT or some other button instead of OK) and then load the list through the menu.

I can only guess that LG never planned the export/import functions to be used for editing a list, but merely to initialize a brand new TV with a list from another identical TV.
